client/src/components/History/Content/model/StatesInfo.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { STATES } from "./states";
+import type { ComputedRef } from "vue";
+import { computed } from "vue";
+
+type State = {
+    status: string;
+    text?: string;
+    icon?: string;
+    spin?: boolean;
+};
+interface States {
+    [key: string]: State;
+}
+interface HelpText {
+    [key: string]: string;
+}
+
+const props = defineProps({
+    showHelp: { type: Boolean, default: false },
+});
+
+const emit = defineEmits<{
+    (e: "update:show-help", showHelp: boolean): void;
+    (e: "set-filter", filter: string, value: string): void;
+}>();
+
+const propShowHelp = computed({
+    get: () => {
+        return props.showHelp;
+    },
+    set: (val) => {
+        emit("update:show-help", val);
+    },
+});
+const states: ComputedRef<States> = computed(() => STATES);
+
+const helpText: HelpText = {
+    failed: "The dataset failed to run.",
```

Comment:
```suggestion
    failed: "The job failed to run.",
```
but again, that's not a dataset state. if that's needed for collections I'd wrangle this to error on the backend.

Reply:
Ok, can change the description for it in the modal